Makefile

```diff
@@ -2,10 +2,22 @@
 all: build
 include tests/robustness/makefile.mk
 
+PLATFORMS=linux-amd64 linux-386 linux-arm linux-arm64 linux-ppc64le linux-s390x darwin-amd64 darwin-arm64 windows-amd64 windows-arm64
```

Comment:
nit: please put `PLATFORMS` right above the target `build-all`

Reply:
Sure, thank you 👍